participants/jonan.json

```
@@ -0,0 +1,18 @@
+{
+    "name": "Jonan Scheffler",
+    "company": "DXOS",
+    "when": {
+        "friday": true,
+        "saturday": true
+    },
+    "iCanTakeNotesDuringSessions": true,
+    "tags": ["JavaScript", "TypeScript", "React"],
+    "whatIsMyConnectionToJavascript": "Relearning modern JavaScript after a few years away.",
+    "whatCanIContribute": "Career coaching, discussion mediation, karaoke thought leadership.",
+    "tShirt": {
+        "size": "3X",
```

Did you mean to set it to `3XL` here?
```suggestion
        "size": "3XL",
```